src/__tests__/userChangemakerPermissions.int.test.ts

```diff
@@ -153,4 +155,206 @@ describe('/users/changemakers/:changemakerId/permissions/:permission', () => {
 			});
 		});
 	});
+
+	describe('DELETE /', () => {
+		it('returns 401 if the request lacks authentication', async () => {
+			const user = await loadTestUser();
+			const changemaker = await createChangemaker({
+				taxId: '11-1111111',
+				name: 'Example Inc.',
+			});
+			await request(app)
+				.delete(
+					`/users/${keycloakUserIdToString(user.keycloakUserId)}/changemakers/${changemaker.id}/permissions/${Permission.MANAGE}`,
+				)
+				.send()
+				.expect(401);
+		});
+
+		it('returns 401 if the authenticated user lacks permission', async () => {
+			const user = await loadTestUser();
+			const changemaker = await createChangemaker({
+				taxId: '11-1111111',
+				name: 'Example Inc.',
+			});
+			await request(app)
+				.delete(
+					`/users/${keycloakUserIdToString(user.keycloakUserId)}/changemakers/${changemaker.id}/permissions/${Permission.MANAGE}`,
+				)
+				.set(authHeader)
+				.send()
+				.expect(401);
+		});
+
+		it('returns 400 if the userId is not a valid keycloak user ID', async () => {
+			await request(app)
+				.delete(
+					`/users/notaguid/changemakers/1/permissions/${Permission.MANAGE}`,
+				)
+				.set(authHeaderWithAdminRole)
+				.send()
+				.expect(400);
+		});
+
+		it('returns 400 if the changemaker ID is not a valid ID', async () => {
+			const user = await loadTestUser();
+			await request(app)
+				.delete(
+					`/users/${keycloakUserIdToString(user.keycloakUserId)}/changemakers/notanId/permissions/${Permission.MANAGE}`,
+				)
+				.set(authHeaderWithAdminRole)
+				.send()
+				.expect(400);
+		});
+
+		it('returns 400 if the permission is not a valid permission', async () => {
+			const user = await loadTestUser();
+			await request(app)
+				.delete(
+					`/users/${keycloakUserIdToString(user.keycloakUserId)}/changemakers/1/permissions/notAPermission`,
+				)
+				.set(authHeaderWithAdminRole)
+				.send()
+				.expect(400);
+		});
+
+		it('returns 404 if the permission does not exist', async () => {
+			const user = await loadTestUser();
+			const changemaker = await createChangemaker({
+				taxId: '11-1111111',
+				name: 'Example Inc.',
+			});
+			await request(app)
+				.delete(
+					`/users/${keycloakUserIdToString(user.keycloakUserId)}/changemakers/${changemaker.id}/permissions/${Permission.MANAGE}`,
+				)
+				.set(authHeaderWithAdminRole)
+				.send()
+				.expect(404);
+		});
+
+		it('returns 404 if the permission had previously been deleted', async () => {
```

Comment:
I wanted to highlight this for consideration.  The internet disagrees on this -- should it be 404 the second time, or 204.  What does idempotent mean in this context?

It's 404 if it never existed of course.

Reply:
@bickelj I wanted to highlight the above comment, since you started to explore some of the questions in it below.  Specifically this question of: `If 204, should it update the not_after to the most recent call, or should not_after not be touched if the permission had already expired?`

Reply:
@jasonaowen @bickelj I do think the above question *is* something we should answer, even without an audit log.

If someone deletes a permission that has already been deleted we should essentially ignore the second request, correct?  Should it return 404, or 204?

Reply:
@slifty I think it should not change `not_after` if it already expired and should return 204 or 404. I'm not certain about the 204 or 404 question, tbh.

Reply:
Yeah, I agree that it should return 404. My reading of this is that "idempotent" means that repeating the same request should not cause issues, and if the server returns 404 on the second request, the state is the same after one or two `DELETE`s: the permission is gone.

Reply:
Thanks y'all!  I'll update to that approach.